### PR TITLE
sync-diff-inspector: update config file

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
1. add config `tidb-instance-id`
2. add config `table-rule`
3. update download urls

### CODE REVIEW

File: tools/sync-diff-inspector.md [hunk 1]
```diff
@@ -223,4 +235,6 @@ password = ""
 
 ### 注意
 
-TiDB 使用的 collation 为 utf8_bin，如果对 MySQL 和 TiDB 的数据进行对比，需要注意 MySQL 中表的 collation 设置。如果表的主键／唯一键为 varchar 类型，且 MySQL 中 collation 设置与 TiDB 不同，可能会因为排序问题导致最终校验结果不正确，需要在 sync-diff-inspector 的配置文件中增加 collation 设置。
\ No newline at end of file
+* TiDB 使用的 collation 为 utf8_bin，如果对 MySQL 和 TiDB 的数据进行对比，需要注意 MySQL 中表的 collation 设置。如果表的主键／唯一键为 varchar 类型，且 MySQL 中 collation 设置与 TiDB 不同，可能会因为排序问题导致最终校验结果不正确，需要在 sync-diff-inspector 的配置文件中增加 collation 设置。
+* 如果设置了 `tidb-instance-id` 使用 TiDB 的统计信息来划分 chunk，需要尽量保证统计信息精确，可以在*业务空闲期*手动执行 `analyze table {table_name}`。
+* table-rule 的规则需要特殊注意，例如设置了 `schema-pattern="test1"`, `target-schema="test2"`，会对比 source 中的 `test1` 库和 target 中的 `test2` 库；如果 source 中有 `test2` 库，该库也会和 target 中的 `test2` 库进行对比。
```
Comment (lilin90) [inline tools/sync-diff-inspector.md]:
```suggestion
* table-rule 的规则需要特殊注意，例如设置了 `schema-pattern="test1"`，`target-schema="test2"`，会对比 source 中的 `test1` 库和 target 中的 `test2` 库；如果 source 中有 `test2` 库，该库也会和 target 中的 `test2` 库进行对比。
```

**Comment by lilin90** [comment]
@csuzhangxc PTAL

File: tools/sync-diff-inspector.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ sync-diff-inspector 是一个用于校验 MySQL／TiDB 中两份数据是否一
 
 GitHub 地址：[sync-diff-inspector](https://github.com/pingcap/tidb-tools/tree/master/sync_diff_inspector)
 
-下载地址：[sync-diff-inspector-linux-amd64.tar.gz](https://download.pingcap.org/sync-diff-inspector-linux-amd64.tar.gz)
+下载地址：[tidb-latest-linux-amd64](https://download.pingcap.org/tidb-latest-linux-amd64.tar.gz)
```
Comment (csuzhangxc) [inline tools/sync-diff-inspector.md]: Any description about we have added sync_diff_inspector into TiDB? Then, the user can know they do not need to download it again.

Comment (lilin90) [reply tools/sync-diff-inspector.md]: @csuzhangxc I mentioned that in this docs PR about Tools Download page: https://github.com/pingcap/docs/pull/855

Comment (csuzhangxc) [reply tools/sync-diff-inspector.md]: @lilin90 in https://github.com/pingcap/docs/pull/855, is *This **enterprise tools** package includes all the following tools:*?

Comment (lilin90) [reply tools/sync-diff-inspector.md]: As far as I know, Yes. @csuzhangxc

Comment (csuzhangxc) [reply tools/sync-diff-inspector.md]: so, whether needing to update description here, because it's *TiDB* not *TiDB Enterprise Tools*?

Comment (csuzhangxc) [reply tools/sync-diff-inspector.md]: or, can we tell the user to get *sync-diff-inspector* from *tidb-enterprise-tools*?

Comment (lilin90) [reply tools/sync-diff-inspector.md]: @GregoryIan What's your opinion, from the _TiDB_ package or the _tidb-enterprise-tools_ package?

Comment (WangXiangUSTC) [reply tools/sync-diff-inspector.md]: I will change to tidb-enterprise-tools, this contains all the tools.

File: tools/sync-diff-inspector.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ sync-diff-inspector 是一个用于校验 MySQL／TiDB 中两份数据是否一
 
 GitHub 地址：[sync-diff-inspector](https://github.com/pingcap/tidb-tools/tree/master/sync_diff_inspector)
 
-下载地址：[sync-diff-inspector-linux-amd64.tar.gz](https://download.pingcap.org/sync-diff-inspector-linux-amd64.tar.gz)
+下载地址：[tidb-latest-linux-amd64](https://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz)
```
Comment (csuzhangxc) [inline tools/sync-diff-inspector.md]: You also need to update `[tidb-latest-linux-amd64]`.

Comment (WangXiangUSTC) [reply tools/sync-diff-inspector.md]: forget update, done

**Pull request merged**